dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1484,6 +1484,7 @@ def refute_bad_gateway_or_site_unreachable
   student_count.to_i.times do |i|
     add_student_step_list.push("Given I create a student named \"student_#{i}\"")
     add_student_step_list.push("And I join the section")
+    add_student_step_list.push("And I wait for 3 seconds")
```

Comment:
Do we know how often the bug affects other tests? Wondering if we should apply the wait globally as part of the `reset_session` logic. This can be a follow up though, looks good!

Reply:
I'm not sure and didn't want to apply that broad of a change unless we were sure we needed it.